OpenTracingBaseTests class can fail due to Maven Central blocking plain http

[OpenTracingBaseTests](https://github.com/eclipse/microprofile-opentracing/blob/master/tck/base/src/main/java/org/eclipse/microprofile/opentracing/tck/OpenTracingBaseTests.java#L85) relies on the default setup that Shrinkwrap does to resolve the opentracing-api artifact.

However since 15/01/2020 Maven Central has started to block plain http requests (see [this](https://support.sonatype.com/hc/en-us/articles/360041287334)) which means that if the Shrinkwrap version doesn't provide proper setup (which is the case for the version used in Quarkus), the tests end up failing with something like:

```
Caused by: java.io.IOException: Server returned HTTP response code: 501 for URL: http://repo1.maven.org/maven2/io/opentracing/opentracing-api/0.31.0/opentracing-api-0.31.0.pom
```

One solution could be to perform some manual setup (perhaps something like):

```
Maven.configureResolver().withMavenCentralRepo(false)
                .withRemoteRepo("Maven Central", "https://repo.maven.apache.org/maven2/", "default");
```

--- a/pom.xml
+++ b/pom.xml
@@ -124,7 +124,7 @@
             <dependency>
                 <groupId>org.jboss.arquillian</groupId>
                 <artifactId>arquillian-bom</artifactId>
-                <version>1.1.13.Final</version>
+                <version>1.6.0.Final</version>
                 <scope>import</scope>
                 <type>pom</type>
             </dependency>

--- a/pom.xml
+++ b/pom.xml
@@ -37,7 +37,7 @@
         <version.mp.rest.client>1.2</version.mp.rest.client>
         <!-- Versions of API dependencies -->
         <osgi-annotation.version>1.0.0</osgi-annotation.version>
-
+        <version.arquillian>1.6.0.Final</version.arquillian>
     </properties>
 
     <licenses>
@@ -124,7 +124,7 @@
             <dependency>
                 <groupId>org.jboss.arquillian</groupId>
                 <artifactId>arquillian-bom</artifactId>
-                <version>1.6.0.Final</version>
+                <version>${version.arquillian}</version>
                 <scope>import</scope>
                 <type>pom</type>
             </dependency>

--- a/tck/pom.xml
+++ b/tck/pom.xml
@@ -33,20 +33,6 @@
     <description>OpenTracing behaviors and APIs for MicroProfile :: TCK</description>
     <packaging>pom</packaging>
     <properties>
-        <arquillian.version>1.1.13.Final</arquillian.version>
         <checkstyle.methodNameFormat>^_?[a-z][a-zA-Z0-9_]*$</checkstyle.methodNameFormat>
     </properties>
-
-    <dependencyManagement>
-        <dependencies>
-            <dependency>
-                <groupId>org.jboss.arquillian</groupId>
-                <artifactId>arquillian-bom</artifactId>
-                <version>${arquillian.version}</version>
-                <type>pom</type>
-                <scope>import</scope>
-            </dependency>
-        </dependencies>
-    </dependencyManagement>
-
 </project>

